src/components/custom-field-input-number/custom-field-input-number.jsx

```diff
@@ -35,6 +35,8 @@ export default function CustomFieldInputNumber(props) {
     setInvalid(!inputRef.current.validity.valid);
   });
 
+  const value = props.value || 0;
```

Comment:
Just a note: we should prefer default props! 

Though, I'm looking into this behavior and I think I would prefer another solution. The reason being: While the UX is near identical, the component now will always have a value. We cannot determine whether a user typed the value `0` or left it `undefined` -- and thus we will always be storing a value in the "backend." I'm gonna timebox some time and see if we can figure out a way to have it as undefined as well.